### PR TITLE
Update mapping parameter to emphasize that it's a json mapping

### PR DESCRIPTION
**Pull request by AsafMah**
Emphasize that the mapping parameter refers to json mapping.

### CODE REVIEW

**Review by yihezkel — CHANGES_REQUESTED** [review]
A few minor verbiage suggestions.

Also, there are additionally some internal references to "mapping" that you didn't rename to json_mapping in ingestor.rb and ingestor_spec.rb. It might be a good idea to make them consistent to reduce dev confusion, even though the Logstash user doesn't interface with them.

File: lib/logstash/outputs/kusto.rb [hunk 1]
```diff
@@ -84,8 +84,9 @@ class LogStash::Outputs::Kusto < LogStash::Outputs::Base
   config :database, validate: :string, required: true
   # Target table name
   config :table, validate: :string, required: true
-  # Mapping name - used by kusto to map an incoming event to the right row format (what value goes into which column)
-  config :mapping, validate: :string
+  # Mapping name - Used by Kusto to map each attribute from incoming event JSON strings to the appropriate column in the table.
+  # Note that this must be in JSON format, as this is the interface between Logstash and Kusto
+  config :json_mapping, validate: :string
```
Comment (tamirkamara) [inline lib/logstash/outputs/kusto.rb:89]: I wanted to point out that this rename is a breaking change and once published will be pulled automatically by any new installation, but will fail existing pipelines config. 
I think you should also note it in the changelog file.

Comment (AsafMah) [reply lib/logstash/outputs/kusto.rb:89]: We move here a major version (0.4.0 -> 1.0.0), shouldn't breaking changes be expected?

Comment (tamirkamara) [reply lib/logstash/outputs/kusto.rb:89]: IMO, no, breaking changes aren't expected.
But this isn't to say you can't introduce them... my comment was that you should document it in the change log.

BTW, there's a way to keep this backward compatible if you wish to make the effort - you can keep the mapping config value and if used assign its value to the new config var. Is this happens, log a warning message about a usage of a deprecated config param.
Not saying you should do the above, but think about it.

**Review by tamirkamara — APPROVED** [review]

**Pull request merged**